webapp/templates/team/submit_modal.html.twig

```diff
@@ -19,21 +19,61 @@
-                        <i class="fas fa-cloud-upload-alt"></i> Submit
-                    </button>
+                    <ul class="nav nav-tabs container text-center" id="submissionTabs" role="tablist" style="width: 100%">
+                        <li class="nav-item" role="presentation" >
+                            <a class="nav-link {% if active_tab == 'upload' %}active{% endif %}" id="upload-tab" data-bs-toggle="tab" href="#upload" role="tab" aria-controls="upload" aria-selected="{% if active_tab == 'upload' %}true{% else %}false{% endif %}" onclick="setCookie('active_tab', 'upload')" >Upload File</a>
+                        </li>
+                        <li class="nav-item text-center" role="presentation">
+                            <a class="nav-link {% if active_tab == 'paste' %}active{% endif %}" id="paste-tab" data-bs-toggle="tab" href="#paste" role="tab" aria-controls="paste" aria-selected="{% if active_tab == 'paste' %}true{% else %}false{% endif %}" onclick="setCookie('active_tab', 'paste')">Paste Code</a>
+                        </li>
+                    </ul>
+                    <div class="tab-content" id="submissionTabsContent" style="margin-top: 20px;">
+                        <!-- File Upload Tab -->
+                        <div class="tab-pane fade {% if active_tab == 'upload' %}show active{% endif %}" id="upload" role="tabpanel" aria-labelledby="upload-tab">
+                            {{ form_start(formupload) }}
+                            {{ form_row(formupload.code) }}
+                            <div class="alert d-none" id="files_selected"></div>
+                            {{ form_row(formupload.problem) }}
+                            {{ form_row(formupload.language) }}
+                            {{ form_row(formupload.entry_point) }}
+                            <div class="modal-footer">
+                                <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
+                                <button type="submit" class="btn btn-success">
+                                    <i class="fas fa-cloud-upload-alt"></i> Submit Upload
+                                </button>
+                            </div>
+                            {{ form_end(formupload) }}
+                        </div>
+
+                        <!-- Paste Code Tab -->
+                        <div class="tab-pane fade {% if active_tab == 'paste' %}show active{% endif %}" id="paste" role="tabpanel" aria-labelledby="paste-tab">
+                            {{ form_start(formpaste) }}
+                            {{ form_widget(formpaste.code_content) }}
+                            <label for="codeInput">Paste your code here:</label>
+                            <div class="editor-container">
+                                {{ "" | codeEditor(
+                                    "_team_submission_code",
+                                    "c_cpp",
```

Comment:
What do you think of moving the language selection above the editor? This way you can also use it here to fix syntax highlighting in the editor (as idea).

Reply:
I think it's a good idea. When I first implemented it, I wasn't sure how to dynamically adjust the syntax highlighting in the editor by changing the language in the menu. Is that achievable?

Reply:
Ah, that depends on the twig template.

Let's leave that for a follow up PR in that case.